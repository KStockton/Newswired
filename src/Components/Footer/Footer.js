import React from 'react';

const Footer = () => {
  
  return (
    <footer className="footer">
      <h5>&copy; News Wire 2019</h5>
    </footer>
  )
}
export default Footer