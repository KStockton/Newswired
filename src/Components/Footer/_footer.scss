.footer {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 2.1rem;
}