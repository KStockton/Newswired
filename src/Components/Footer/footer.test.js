import React from 'react';
import Footer from './Begin';
import { shallow } from 'enzyme';

describe('Footer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
});