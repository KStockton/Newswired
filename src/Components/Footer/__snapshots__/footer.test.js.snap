// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<footer
  className="footer"
>
  <h5>
    Â© News Wire 2019
  </h5>
</footer>
`;
